weave/weave_client.py

```diff
@@ -462,7 +462,7 @@ def create_call(
         if parent is not None:
             parent._children.append(call)
 
-        current_wb_run_id = safe_current_wb_run_id()
```

Comment:
refactor for clarity, this is a URI not an id

Reply:
It's not a URI though - to be a URI it would have to start with a scheme, looking at the implementation it is just `<entity>/<project>/<run_id>`

Reply:
Right, sorry, reccomendations on what we should call it? I could also type it as an object @jamie-rasmussen

Reply:
I'd probably just leave it named as it was - even though just calling it `wb_run_id` is a little inaccurate, I think that's consistent with what is called in a bunch of places.